components/brave_ads/content/browser/creatives/search_result_ad/creative_search_result_ad_mojom_web_page_entities_extractor_unittest.cc

```diff
@@ -236,13 +260,9 @@ TEST(BraveAdsCreativeSearchResultAdMojomWebPageEntitiesExtractorTest,
   {
     // Invalid "data-rewards-value".
     const std::vector<schema_org::mojom::EntityPtr> mojom_web_page_entities =
-        test::CreativeSearchResultAdMojomWebPageEntities(
-            /*excluded_property_names=*/{kCreativeAdRewardsValuePropertyName});
-    const auto& mojom_property = mojom_web_page_entities[0]->properties[0];
-    auto& mojom_entity = mojom_property->values->get_entity_values()[0];
-    test::AddMojomProperty<std::string>(&mojom_entity->properties,
-                                        kCreativeAdRewardsValuePropertyName,
-                                        /*value=*/"0-5");
+        test::CreativeSearchResultAdMojomWebPageEntitiesWithProperty(
+            /*property_name=*/kCreativeAdRewardsValuePropertyName,
```

Comment:
nit: no need for `/*property_name=*/` because the constant ends with `PropertyName`